api/DummyData/getAllUsersFuncReturn.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-vars */
+const getAllUsersReturn = [
```

Comment:
As per [my previous comment](https://github.com/bkarimii/slack-dashboard/pull/10/files#r1922808532), classically, a constant like this would be called `USERS`